src/Reflex/Dom/Xhr.hs

```diff
@@ -5,12 +5,14 @@
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TupleSections #-}
```

Comment:
I'd prefer to avoid TupleSections if it's not too big a change.  In my experience, it has a tendency to turn typos into very difficult to understand error messages.